content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/owned_stock.rst

```diff
@@ -74,9 +74,9 @@ quotation is complete, click :guilabel:`Confirm`.
    :align: center
    :alt: Sales order of consignment stock.
 
-Once the :abbr:`RFQ (Request for Quotation)` has been confirmed, it will become a sales order. From
-here, the products can be delivered by clicking on the :guilabel:`Delivery` smart button, and
-selecting :guilabel:`Validate` to validate the delivery.
+Once the :abbr:`Quotation` has been confirmed, it will become a sales order. From here, the products
+can be delivered by clicking on the :guilabel:`Delivery` smart button, and selecting
+:guilabel:`Validate` to validate the delivery.
```

Comment:
Please remove the abbr tag from the non-abbreviated word and remove future tense :)
```suggestion
Once the quotation has been confirmed, it becomes a sales order. From here, the products can be
delivered by clicking on the :guilabel:`Delivery` smart button, and selecting :guilabel:`Validate`
to validate the delivery.
```